modules/openlmis-web/src/main/java/org/openlmis/web/util/FacilityDistributionEditHandler.java

```diff
@@ -180,25 +185,51 @@ private void checkProperties(FacilityDistributionEditResults results, Object par
           originalProperty = removeNullReference(original, originalPropertyName, originalProperty);
           replacementProperty = removeNullReference(replacement, replacementPropertyName, replacementProperty);
 
-          if (originalProperty instanceof List) {
+          if (originalProperty instanceof List)
```

Comment:
Would you mind re-adding the "{" please? I know it's a trivial request. OpenLMIS v3 enforces this stylistic standard, though, and it would be nice if we followed it for consistency.